node/bundler.ts

```diff
@@ -173,18 +172,31 @@ const getBasePath = (sourceDirectories: string[], inputBasePath?: string) => {
   return commonPathPrefix(sourceDirectories)
 }
 
-export const addGeneratorFieldIfMissing = (
-  declaration: Declaration,
-  functions: EdgeFunction[],
-  internalFunctionsPath?: string,
-) => {
-  const fullFuncPath = functions?.find((func) => func.name === declaration.function)?.path
-
+const addMissingConfigFields = ({
```

Comment:
I don't think "missing config fields" is very descriptive — what's missing and why? Perhaps we can add a comment explaining why we need this function?

Reply:
I decided to switch it up a bit and make it, hopefully, more descriptive, and give functions a clear single task.